### ISSUE TITLE
Fix importing of kotlin on gradle

### ISSUE THREAD

**Issue by colinrtwhite** [snowplow/snowplow-android-tracker#396]
This project is written 100% in Java, however the SDK ships with a dependency on [the Kotlin stdlib](https://github.com/snowplow/snowplow-android-tracker/blob/master/snowplow-tracker/build.gradle#L85) and [Kotlin Android extensions](https://github.com/snowplow/snowplow-android-tracker/blob/master/snowplow-tracker/build.gradle#L7). Kotlin was added in [this PR](https://github.com/snowplow/snowplow-android-tracker/pull/358), but seems unrelated?

Also as an aside it would be great if this library added nullability annotations to make Kotlin interoperability nicer! I can open up a separate issue for this if you'd prefer.

**Comment by AlexBenny** [comment]
Hi @colinrtwhite , thanks for reporting this.
We will remove the unnecessary dependency in the next release coming soon.
I think we added them for few experiments and then mistakenly shipped on the release.

About the nullability annotation we are keen to add them for our two native mobile trackers (iOS and Android) with other API improvements. It's already planned for the version 2.0 which should be released in the coming months.
We added the annotation to few methods and classes we have refactored recently but it doesn't cover the full API at the moment.
